lib/rpi_fb_capture.ex

```diff
@@ -76,6 +76,14 @@ defmodule RpiFbCapture do
     GenServer.call(server, {:mono_threshold, threshold})
   end
 
+  @doc """
+  Enable/disable dithering
+  """
+  @spec set_dithering(GenServer.server(), byte()) :: :ok | {:error, atom()}
+  def set_dithering(server, enabled?) do
```

Comment:
It's been a while since I've had to dither images, but last time I remember there being a discussion on algorithms since Floyd-Steinberg gave less desirable results for some images. How about making the API something like this:

```elixir
@spec set_dithering(GenServer.server(), :none | :floyd_steinberg) :: :ok | {:error, atom()}
def set_dithering(server, dither) do
```

Reply:
The intention is to let someone extend it to include another dithering method without breaking existing code.

Reply:
I like it! I'll update the PR to support that